<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>641</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>19</number>
    </property>
    <item>
     <widget class="QLabel" name="lblMundo">
      <property name="text">
       <string>MUNDO ( 500, 500 )</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="zoomMenos">
      <property name="text">
       <string>( - )</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="zomMais">
      <property name="text">
       <string>( + )</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblViewPort">
      <property name="text">
       <string>VIEW PORT ( -250, -250 )</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>7</y>
     <width>681</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="4">
     <widget class="QPushButton" name="btnCasteljau">
      <property name="text">
       <string>CASTELJAU</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnPoligono">
      <property name="text">
       <string>POLIGONO</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnHermit">
      <property name="text">
       <string>HERMIT</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QListWidget" name="listPoligonos">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnBezier">
        <property name="text">
         <string>BEZIER</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnEscalonar">
        <property name="text">
         <string>ESCALONAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ESCALA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbEsc">
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btnExcluir">
      <property name="text">
       <string>EXCLUIR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnClipping">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>CLIPPING</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QListWidget" name="listPontos"/>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnRotaciona">
        <property name="text">
         <string>ROTACIONAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbR">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbtNorm">
        <property name="text">
         <string>NORMAL</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbtHomo">
        <property name="text">
         <string>HOMOGÊNIA</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>LISTA DE PONTOS</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnCirculo">
      <property name="text">
       <string>CÍRCULO</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnTranslada">
        <property name="text">
         <string>TRNSLADAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbTx">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbTy">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>LISTA DE POLÍGONOS E CIRCUNFERÊNCIAS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>125</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="radiobtnAlgoritmo">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QRadioButton" name="rdbtDda">
      <property name="text">
       <string>DDA</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdbtBresiham">
      <property name="text">
       <string>Bresiham</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdbtQt">
      <property name="text">
       <string>Qt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
